.junie/plans/diagnose-database-wipe.md

```
@@ -0,0 +1,374 @@
+---
+sessionId: session-260824-172958-1864
+---
+
+# Requirements
+
+### Overview & Goals
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)